website/content/docs/k8s/deployment-configurations/single-dc-multi-k8s.mdx

```diff
@@ -147,6 +163,8 @@ NAME                 TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)         AGE
 cluster1-consul-ui   NodePort   10.0.240.80   <none>        443:31557/TCP   40h
 ```
 
+The `grpcPort: 30014` refers to the grpc nodePort set in the first cluster.
```

Comment:
```suggestion
The `grpcPort: 30014` configuration refers to the gRPC port number specified in the `NodePort` configuration in the first cluster.
```
It's `NodePort` and not `nodePort`, correct?

Reply:
@trujillo-adam that's correct, it should be `nodePort`  👍